typescript/cli/src/commands/registry.ts

```diff
@@ -71,19 +72,34 @@ const listCommand: CommandModuleWithContext<{ type: ChainType }> = {
 /**
  * Addresses command
  */
-const addressesCommand: CommandModuleWithContext<{ name: string }> = {
+const addressesCommand: CommandModuleWithContext<{
+  name: string;
+  contract: string;
+}> = {
   command: 'addresses',
+  aliases: ['address', 'addy'],
   describe: 'Display the addresses of core Hyperlane contracts',
   builder: {
     name: {
```

Comment:
first time seeing this command, but kinda weird that the arg is "name" and "chain" is it's alias

Reply:
I believe Kunal requested this alias a while back

Reply:
agree would have probably swapped this but I dont think that is material to the UX